src/Tippy.Core/Address/Address.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Tippy.Util;
+using Types = Ckb.Rpc.Types;
+
+namespace Tippy.Core.Address
+{
+    public class Address
+    {
+        static readonly string SECP_CODE_HASH = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8";
```

Comment:
Please follow dot net naming convention. Constant (field) should use `CamelCase`.

Reply:
updated